src/osgEarthDrivers/gdal/ReaderWriterGDAL.cpp

```diff
@@ -2592,6 +2608,7 @@ class GDALTileSource : public TileSource
     double       _invtransform[6];
     double       _linearUnits;
 
+    std::vector<std::tuple<double, double, double, double>> _tilesExtent;
```

Comment:
The fact that a structure is not used make it hard to understand the meaning (or at least the order) of the tuple.
So please add comment '(minY, maxY, minX, maxX)' everywhere needed

Reply:
Or maybe use Bounds type if exported